.github/linters/codespell.txt

```diff
@@ -211,6 +216,7 @@ faield
 faild
 failes
 falied
+fals
```

Comment:
I can only see one usage of `fals` which is in `plugins/hypervisors/hyperv/DotNet/ServerResource/WmiWrappers/ROOT.virtualization.v2.Msvm_ExternalEthernetPort.cs`. Better fix in the file and remove `fals` from here.